cluster-install/install.sh

```diff
@@ -1351,13 +1366,19 @@ step_check_config() {
     add_to_config_builder "NAMESPACE"
 
     if prompts_disabled; then
-        if [ -z "$SKIP_TRIDENT_UPGRADE" ]; then
-            local -r longer_msg="SKIP_TRIDENT_UPGRADE is required when prompts are disabled."
-            add_problem "SKIP_TRIDENT_UPGRADE: required (prompts disabled)" "$longer_msg"
+        if [ -z "$DO_NOT_MODIFY_EXISTING_TRIDENT" ]; then
+            local -r longer_msg="DO_NOT_MODIFY_EXISTING_TRIDENT is required when prompts are disabled."
+            add_problem "DO_NOT_MODIFY_EXISTING_TRIDENT: required (prompts disabled)" "$longer_msg"
         fi
     fi
+    if [ "$DO_NOT_MODIFY_EXISTING_TRIDENT" == "true" ] && [ "$COMPONENTS" == "$__COMPONENTS_ACP_ONLY" ]; then
+        local -r acp_only_err="DO_NOT_MODIFY_EXISTING_TRIDENT: cannot be set to 'true' if using"
+        acp_only_err+=" COMPONENTS='${__COMPONENTS_ACP_ONLY}', as the component in question requires modifying an"
+        acp_only_err+=" existing Trident installation."
```

Comment:
`DO_NOT_MODIFY_EXISTING_TRIDENT` can't be set to **true** if `COMPONENTS=ACP_ONLY` because that operation, by its very nature, requires modifying an existing Trident install.

`TRIDENT_ONLY` is allowed because it can still mean doing a fresh install. There is a check later on to see if there is an existing Trident install with TRIDENT_ONLY set and DO_NOT_MODIFY_EXISTING_TRIDENT=true, at which point we will error out of the script since that's also incompatible (we just can't know about it until we collect trident info from the cluster).